modules/nw-sriov-operator-uninstall.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/uninstalling-sriov-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-operator-uninstall_{context}"]
+= Uninstalling the SR-IOV Network Opreator
+
+As a cluster administrator, you can uninstall the SR-IOV Network Operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have the SR-IOV Network Operator installed.
+
+.Procedure
+
+. Delete all SR-IOV custom resources (CRs):
++
+[source,terminal]
+----
+$ oc delete sriovnetwork -n openshift-sriov-network-operator --all
+----
++
+[source,terminal]
+----
+$ oc delete sriovnetworknodepolicy -n openshift-sriov-network-operator --all
+----
++
+[source,terminal]
+----
+$ oc delete sriovibnetwork -n openshift-sriov-network-operator --all
+----
+
+. Follow the instructions on xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting operators from a cluster] to remove the SR-IOV Network Operator from your cluster.
```

Comment:
We're not allowed to have xrefs in modules, so you'll have to rethink how you do this step. You could just reference the section title here, like this:

```
Follow the instructions in the "Deleting Operators from a cluster" section to remove the SR-IOV Network Operator from your cluster. 
```
(note that Operator is capitalized)

... and then add the xref in the assembly in an ".Additional resources" section after you include this new module. Let me know if you need an example of how to do that!

Reply:
This is how you would do the additional resources section in the assembly:

```
...

include::modules/nw-sriov-operator-uninstall.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

* xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster]
```

Reply:
Thank you for sharing an example!